lib/translation_io/client/base_operation.rb

```diff
@@ -40,14 +40,14 @@ def self.perform_request(uri, params)
             return parsed_response
           elsif response.code.to_i == 400 && parsed_response.has_key?('error')
             $stderr.puts "[Error] #{parsed_response['error']}"
-            exit(false)
+            return nil
```

Comment:
Update return of these to nil so the outter task can iterate over multiple domains